frontend/src/app/state/hierarchy.actions.ts

```diff
@@ -71,32 +72,62 @@ export const deleteHierarchyFailure = createAction(
     props<{ error: string }>()
 );
 
+export const createAlternative = createAction(
+    '[Alternatives] Create Hierarchy',
+    props<{ createAlternativeForm: CreateAlternativeForm }>()
```

Comment:
Probably best to give this prop a different name since the store doesn't care about "forms".